File: black-note-server/springboot-redis/src/main/java/com/syr/entity/ChatSession.java
package com.syr.entity;

import com.baomidou.mybatisplus.annotation.TableName;
import lombok.Data;

import java.time.LocalDateTime;

@Data
@TableName("chat_sessions")
public class ChatSession {
    private String id;
    private Long   userId;
    private String title;
    private LocalDateTime createdAt;
    private LocalDateTime updatedAt;
}

File: black-note-ai/app/schemas/chat_session.py
from datetime import datetime
from pydantic import BaseModel


class ChatSessionUpsertRequest(BaseModel):
    title: str = "新对话"


class ChatSessionResponse(BaseModel):
    session_id: str
    user_id:    str
    title:      str | None = None
    created_at: datetime | None = None
    updated_at: datetime | None = None
